cpp/bench/ann/CMakeLists.txt

```diff
@@ -51,7 +51,31 @@ option(CUVS_KNN_BENCH_USE_CUVS_BRUTE_FORCE "Include cuVS brute force knn in benc
 
 find_package(Threads REQUIRED)
 
-set(CUVS_ANN_BENCH_USE_FAISS ON)
+# ----- FAISS use in Benchmarks ----
+get_cmake_property(_variableNames VARIABLES)
+
+set(CUVS_ANN_BENCH_USE_FAISS OFF)
+message(STATUS "Checking for FAISS use in benchmarks...")
+foreach(_varName ${_variableNames})
+    if(_varName MATCHES "CUVS_ANN_BENCH_USE_FAISS.+")
+        if(DEFINED ${_varName})
```

Comment:
We are looping through the list of defined VARIABLEs, is this condition necessary?